troubadix/plugins/dependencies.py

```diff
@@ -72,7 +74,9 @@ def run(
                     if "+d+.nasl" in dep:
                         continue
 
-                    if not (root / dep).exists():
+                    if not any(
+                        (root / vers / dep).exists() for vers in FEED_VERSIONS
+                    ):
```

Comment:
Maybe I'm wrong, but when the feed is generated it's all in one directory called nasl. Maybe check (root / dep) too?

Reply:
i thought about this, but this script is not running there, I think?

Reply:
At least on every PR it is running on a generated feed release.

Reply:
I have added `""` ;)